gatsby-config.js

```diff
@@ -121,5 +115,11 @@ module.exports = {
         icon: 'src/images/icon.svg',
       },
     },
+    {
+      resolve: `gatsby-transformer-remark`,
+      options: {
+        plugins: [`gatsby-remark-smartypants`],
```

Comment:
The new plugin should just be added to the prior entry for gatsby-transformer-remark entry in line 86

Reply:
@kevee Eep, I totally missed that—thank you! How’s da5d778206bb30ae54bf9ba934b9532b9bde232e look to you?